Demo/Sources/Showcase/SourceCodeViewable.swift

```diff
@@ -44,7 +46,8 @@ extension View {
     private func gitHubUrl<T>(for objectType: T.Type) -> URL? where T: SourceCodeViewable {
         let separator = "/Demo/"
         guard let relativePath = objectType.filePath.split(separator: separator).last else { return nil }
-        var url = URL("github://github.com/SRGSSR/pillarbox-apple/blob/main")
+        var url = URL("github://github.com/SRGSSR/pillarbox-apple/blob")
+            .appending(component: Player.version)
```

Comment:
Doesn't `Player.version` return something like `0.6.0-32-xxxxxx` for nightly releases?
In this case, the link won't work I think.

Reply:
it works, and you can try the following link:

https://github.com/SRGSSR/pillarbox-apple/blob/Player%200.6.0-32-g3f1669a8/Demo/Sources/Showcase/BlurredView.swift